src/config/loadGitHubAppsFromEnvironment.ts

```diff
@@ -58,12 +63,32 @@ export class GitHubAppsRegistry {
 }
 
 export function loadGitHubAppsFromEnvironment(env) {
-  const apps = new GitHubAppsRegistry();
+  const configs = new GitHubAppsConfigHelper();
+  let config;
 
   if (env.GH_APP_IDENTIFIER && env.GH_APP_SECRET_KEY) {
-    const org = '__tmp_org_placeholder__';
-    apps.register(new GitHubApp(org, env));
+    // Collect config by (proleptic) envvar number. Once we have GH_APP_1_FOO
+    // this will make more sense. We'll collect shtuff in config and then
```

Comment:
```suggestion
    // this will make more sense. We'll collect stuff in config and then
```

Reply:
I was being silly. 😛

Merged (it's rewritten anyway in #524).